fgrpc/pingsrv.go

```diff
@@ -65,14 +65,10 @@ func (s *pingSrv) Ping(c context.Context, in *PingMessage) (*PingMessage, error)
 }
 
 func shouldInjectError(errorRate int) bool {
-	rand.Seed(time.Now().UnixNano())
-	number := rand.Intn(100 + 1)
-	fail := false
-
-	if number < errorRate {
-		fail = true
+	if int(100.*rand.Float32()) < errorRate {
```

Comment:
As I mentioned, global rand functions are not recommended to use in prod.
It might be ok for now, but it's better to have a package var for rand, init with a seed (in init() func), and then use it with a mutex.

Also, I would change errorRate type as float32 so that you can compare with it directly without multiplication and type conversion.